src/DacpacTool/DacpacTool.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net9.0;net8.0</TargetFrameworks>
```

Comment:
nit: it doesn't matter at all, but it bugs me that these are out of order :slightly_smiling_face: